docs/diagrams/ModelClassDiagram.puml

```diff
@@ -16,6 +16,7 @@ Class UniquePersonList
 Class Person
```

Comment:
Might need to consider using other line types or hidden lines to make the diagram less cluttered